config.json

```diff
@@ -0,0 +1 @@
+{ "accessKeyId": "AKIAJLFCQN3TGR5OQOFA", "secretAccessKey": "ZjV//dYSifhA+qGFWKr/dfkanuhEdWJiIWAa/8ys", "region": "us-east-1" }
```

Comment:
You may not want to commit your config file with access keys. Add it to .gitignore.